Ryujinx.Graphics.Nvdec.FFmpeg/Native/FFmpegApi.cs

```diff
@@ -78,52 +78,52 @@ static FFmpegApi()
 
         public unsafe delegate void av_log_set_callback_callback(void* a0, AVLog level, [MarshalAs(UnmanagedType.LPUTF8Str)] string a2, byte* a3);
 
-        [DllImport(AvUtilLibraryName, CallingConvention = CallingConvention.Cdecl)]
-        internal static unsafe extern AVFrame* av_frame_alloc();
+        [LibraryImport(AvUtilLibraryName)]
+        internal static unsafe partial AVFrame* av_frame_alloc();
```

Comment:
```suggestion
        [UnmanagedCallConv(CallConvs = new[] { typeof(CallConvCdecl) })]
        internal static unsafe partial AVFrame* av_frame_alloc();
```
This is probably not necessary, since it also seems to work without that. So you could just leave it like that.